privacy/enforcement.go

```diff
@@ -60,12 +75,11 @@ func (e Enforcement) getUserScrubStrategy(ampGDPRException bool) ScrubStrategyUs
 		return ScrubStrategyUserIDAndDemographic
 	}
 
-	if e.GDPR && ampGDPRException {
-		return ScrubStrategyUserNone
+	if e.CCPA || e.LMT {
```

Comment:
This looks like a bug too? If CCPA is enforced along with GDPR for a AMP request, the user would not be scrubbed. Now the AMP exception applies only to the GDPR enforcement aspect.